VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/data/Credentials.java

```diff
@@ -1,11 +1,26 @@
 package org.edx.mobile.test.feature.data;
 
+import android.support.annotation.Nullable;
+import org.edx.mobile.test.feature.config.E2EConfig;
+import org.edx.mobile.util.ResourceUtil;
+
+import java.util.UUID;
+
 public class Credentials {
     public final String email;
     public final String password;
+    @Nullable  public String username;
```

Comment:
Instead of making this nullable, should we just do a slightly bigger factor and require a username? It's needed in practice anyway